docker-entrypoint.sh

```diff
@@ -4,6 +4,9 @@ set -e
 if [[ "$1" == "lnd" || "$1" == "lncli" ]]; then
 	mkdir -p "$LND_DATA"
 
+    # removing noseedbackup=1 flag, adding it below if needed for legacy
+    LND_EXTRA_ARGS=${LND_EXTRA_ARGS/noseedbackup=1/}
```

Comment:
I guess this means you can't intentionally create an lnd instance with `noseedbackup` anymore? Not that it is an issue..

Reply:
Yeah, that init/unlock script depends on noseedbackup=1 not being. We can have something like noseedbackup=special_value_1 ... but I doubt anyone would use it. They would just go with this auto init/unlock = same behavior.